website/static/js/pages/project-settings-page.js

```diff
@@ -252,5 +252,37 @@ $(document).ready(function() {
             }
         }
     });
-
 });
+
+var WikiSettingsViewModel = {
+    enabled: ko.observable(ctx.wiki.isEnabled), // <- this would get set in the mako template, as usual
+    wikiMessage: ko.observable('')
+};
+
+var self = this;
```

Comment:
No need to attach to `this` here; it does not serve a purpose. You should just be able to do

```js
WikiSettingsViewModel.enabled.subscribe(...)
```